test/main_test.go

```diff
@@ -281,3 +290,76 @@ func deleteClusterWideTestOperators() {
 		}
 	}
 }
+
+func monitorOlmNamespace() {
+	monitorNamespace(config.GetOlmNamespace())
+}
+
+func monitorKogitoOperatorNamespace() {
+	monitorNamespace(kogitoClusterWideNamespace)
+}
+
+func monitorNamespace(namespace string) {
+	go func() {
+		err := framework.StartPodLogCollector(namespace)
+		if err != nil {
+			framework.GetLogger(namespace).Errorf("Error starting log collector for namespace %s: %v", namespace, err)
+		}
+	}()
+}
+
+func stopOlmNamespaceMonitoring() {
+	stopNamespaceMonitoring(config.GetOlmNamespace())
+}
+
+func stopKogitoOperatorNamespaceMonitoring() {
+	stopNamespaceMonitoring(kogitoClusterWideNamespace)
+}
+
+func stopNamespaceMonitoring(namespace string) {
+	if err := framework.StopPodLogCollector(namespace); err != nil {
+		framework.GetMainLogger().Errorf("Error stopping log collector on namespace %s: %v", namespace, err)
+	}
+	if err := framework.BumpEvents(namespace); err != nil {
+		framework.GetMainLogger().Errorf("Error bumping events for namespace %s: %v", namespace, err)
+	}
+}
+
+// Install cluster wide Kogito operator
+func installClusterWideKogitoOperator() error {
+	// Check that the kogito operator namespace doesn't exist yet
+	if exists, err := framework.IsNamespace(kogitoClusterWideNamespace); err != nil {
+		return err
+	} else if exists {
+		return fmt.Errorf("Namespace %s used to deploy cluster wide Kogito operator exists, Kogito operator may be already installed. Please uninstall the operator first", kogitoClusterWideNamespace)
+	}
+
+	// Create the namespace
+	if err := framework.CreateNamespace(kogitoClusterWideNamespace); err != nil {
+		return err
+	}
+
+	// Deploy the operator
+	created, err := framework.DeployClusterWideKogitoOperatorFromYaml(kogitoClusterWideNamespace)
+	clusterObjects = created
+	if err != nil {
+		return err
+	}
+
```

Comment:
here we are not making sure that the Kogito Operator is running ?
if we don't, that will lead to errors in all scenarios or ?

Reply:
and then we should not need to wait for operator if cluster wide
- https://github.com/kiegroup/kogito-cloud-operator/pull/694/files#diff-87ffc708dd3b79b9732e75109db3f4ff2066ebb4d9e1dfb4964d4625710f7674R112-R114
- https://github.com/kiegroup/kogito-cloud-operator/pull/694/files#diff-87ffc708dd3b79b9732e75109db3f4ff2066ebb4d9e1dfb4964d4625710f7674R61-R63

Reply:
Will add the waiting code. The links above don't work :/

Reply:
I was meaning:
- https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/steps/operator.go#L54-L56
- https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/steps/operator.go#L100-L102
that goes with my comment on problems => https://github.com/kiegroup/kogito-cloud-operator/pull/694#issuecomment-742684258

Reply:
Now the Kogito operator waiting in scenario steps should happen only for namespace scoped deployment.